packages/ui-module-header/src/components/profile.tsx

```diff
@@ -0,0 +1,47 @@
+import Image from 'next/image';
+import { ProfileProps } from '../types';
+import { Dropdown, Icon } from '@test-assessment/ui-components';
+import React from 'react';
+
+export const Profile = (props: ProfileProps) => {
+  const { name, image } = props;
+  const [open, setOpen] = React.useState(false);
+  const styles = useStyles();
+
+  return (
+    <Dropdown
+      button={
+        <div className={`${styles.root}`}>
+          <Image src={image} alt={''} />
```

Comment:
Like in design: https://www.figma.com/file/TVnPZvRLFvDrVhDMloLiLS/L%26D?type=design&node-id=925-43687&t=KMW8HmR3SNp2DsVe-4
We have a default avatar, so we need to add the default in here, for example: 
` <Image src={image??'./images/avatar.png'} alt={''} />`